benchmarking/frameworks/caffe2/caffe2.py

```diff
@@ -179,7 +179,7 @@ def _processDelayData(self, data):
         for d in data:
             for k, v in d.items():
                 details[k]["values"].append(v)
-        pattern = re.compile(r"^ID_(\d+)_([a-zA-Z]+)_[\w/]+")
+        pattern = re.compile(r"^ID_(\d+)_([a-zA-Z0-9]+)_[\w/]+")
```

Comment:
Maybe when the pattern doesn't match, make sure it is graph timing, and assert otherwise.